nservicebus/upgrades/support-policy.md

```diff
@@ -11,9 +11,11 @@ related:
 
 Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
 
-If a version is not listed or has expired but support is required [contact Particular](https://particular.net/contactus). For more information on the relationship between releases and support see [the release policy](/nservicebus/upgrades/release-policy.md).
+Particular guarantees patching all supported versions with critical bug fixes, even if the specific issue issue was raised only for one version. In case the bug also impacts non supported version, the recommended approach is to upgrade to the supported version which contains the fix.
```

Comment:
>even if the specific issue issue was raised only for one version

Do we need to say this? The previous statement already says that we "even if the specific issue issue was raised only for one version" so if the issue fits that it goes without saying that we'll do it?